src/wdio/services/AxeService.js

```diff
@@ -33,7 +33,7 @@ export default class AxeService {
       const specifiedViewports = options.viewports || [currentViewportSize];
       const axeOptions = {
         runOnly: options.runOnly,
-        rules: options.rules,
+        rules: Object.assign({}, axeConfig.rules, options.rules),
```

Comment:
the global options should be set once upon initialization of the script by appending that config block to line 24

Reply:
append the json config to the javascript source code that is read in on lines 23 and 24?

```
axeCoreSrc = fs.readFileSync(require.resolve('axe-core'), 'utf8');
axeCoreSrc = axeCoreSrc.replace(/^\/\*.*\*\//, '');
```

Reply:
yeah